udm/beginner/party.md

```diff
@@ -79,6 +79,91 @@ Sample JSON Data
 }
 ```
 
+<br>
```

Comment:
Instead of duplicating the JSON data, could you make the necessary changes directly in the original JSON?